### ISSUE TITLE
DiamondOperatorForVariableDefinitionCheck throws NullPointerException

### ISSUE THREAD

**Issue by stephanenicolas** [sevntu-checkstyle/sevntu.checkstyle#348]
When running 'com.github.sevntu.checkstyle:sevntu-checks:1.13.0', we get a stack trace : 
```
java.lang.NullPointerException
	at com.github.sevntu.checkstyle.checks.coding.DiamondOperatorForVariableDefinitionCheck.isSameTypeArgsInVariableDef(DiamondOperatorForVariableDefinitionCheck.java:86)
	at com.github.sevntu.checkstyle.checks.coding.DiamondOperatorForVariableDefinitionCheck.visitToken(DiamondOperatorForVariableDefinitionCheck.java:65)
	at com.puppycrawl.tools.checkstyle.TreeWalker.notifyVisit(TreeWalker.java:393)
	at com.puppycrawl.tools.checkstyle.TreeWalker.processIter(TreeWalker.java:482)
	at com.puppycrawl.tools.checkstyle.TreeWalker.walk(TreeWalker.java:322)
```
The problem comes from :
````java
    private static boolean isSameTypeArgsInVariableDef(DetailAST variableDefNode,
            DetailAST typeArgs) {

        DetailAST typeNode = variableDefNode.findFirstToken(TokenTypes.TYPE);
        DetailAST variableDefTypeArgs = typeNode.findFirstToken(TokenTypes.TYPE_ARGUMENTS);
        
        return variableDefTypeArgs.equalsTree(typeArgs);        // <---     variableDefTypeArgs is null
    }
```

**Comment by stephanenicolas** [comment]
Btw, we have a lot of constructs using 

`Type<A> t = new Type<A> {}`, that might be where it fails.

**Comment by romani** [comment]
@stephanenicolas , thanks a lot. 

If you can send share with us exact file - it will be great help, we will recheck code chunk that you have provided.

**Comment by romani** [comment]
@stephanenicolas , we already have that case in our UTs - https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/master/sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/InputDiamondOperatorForVariableDefinitionCheck.java#L62 , works fine.

please provide exact file to reproduce a problem, you can obfuscate code if you afraid to share source.

**Comment by romani** [comment]
reproduced on code:
```Class<?>[] resolvedInterfaces = new Class<?>[5];```

**Comment by stephanenicolas** [comment]
Thx guys, I couldn't have shared the code.

**Comment by romani** [comment]
fixed, released as 1.13.2
